### ISSUE TITLE
Can you remove error message when debug by type 'ahk'?

### ISSUE THREAD

**Issue by cweijan** [zero-plusplus/vscode-autohotkey-debug#18]
I made some adjustments, and now when the extension [ahk-plus](https://github.com/cweijan/vscode-autohotkey) clicks the run button, can start your debugger correctly, which means that both extensions can be installed at the same time.
![image](https://user-images.githubusercontent.com/27798227/86194037-0ad28c00-bb80-11ea-86b5-6911b57ea6b4.png)

**Comment by SAbboushi** [comment]
What is benefit of installing both extensions at same time?  What features does your extension offer that is not in vscode-autohotkey-debug?

**Comment by cweijan** [comment]
@SAbboushi  This is the extension that currently provides the most feature for the ahk language.

**Comment by zero-plusplus** [comment]
In conclusion, the error message cannot be removed.

In the current version `ahk` is deprecated and will not be debuggable in future updates.
The reason I changed it is that `ahk` looks like a file extension. (Especially from those who use AutoHotkey v2)
Therefore, I chose `autohotkey` to reduce misunderstandings.

So there is nothing I can do.

But there are ways to solve the problem.
If you look at [here](https://github.com/cweijan/vscode-autohotkey/blob/master/src/core/ScriptRunner.ts#L15), debugging is started with fixed settings.  Using the launch.json configuration to start debugging should solve the problem I think so.
